packages/qwik/src/core/v2/client/vnode-diff.ts

```diff
@@ -702,7 +721,7 @@ export const vnode_applyJournal = (journal: VNodeJournalEntry[]) => {
         while (typeof (key = journal[idx] as string | null) === 'string') {
           idx++;
           const value = journal[idx++] as string | null;
-          if (key.startsWith(':')) {
+          if (key.startsWith(':') || key.startsWith('document:') || key.startsWith('window:')) {
```

Comment:
So this is not right. The code needs to stay as is, which means the code above needs to change.

The way it works now is that any property that starts with `:` is not serializable. 

For example `:onClick$` So i think we need to have `:document:onClick$` or something like that.